Guided Investigation - Incident Triage.ipynb

```diff
@@ -0,0 +1,1470 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    start = pytz.utc.localize(alert_q_times.start)
</pre><p>Are you localizing the times for user convenience. Maybe worth noting that somewhere. It might be confusing if people are used to working in UTC</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/120/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>